src/dcp/dcp.c

```diff
@@ -65,7 +65,9 @@ void print_usage(void)
 #ifdef LUSTRE_SUPPORT
     /* printf("  -g, --grouplock <id> - use Lustre grouplock when reading/writing file\n"); */
 #endif
+    printf("  -b, --blocksize     - IO buffer size in MB (default 1MB)\n");
     printf("  -i, --input <file>  - read source list from file\n");
+    printf("  -k, --chunksize     - work size per task in MB (default 1MB)\n");
```

Comment:
According to the comment about mfu_abtoull, let's update this and the blocksize descriptions to be "work size per task in bytes (default 1MB)"

Reply:
This wouldn't be needed with the above change.